python/coinbase-agentkit/coinbase_agentkit/action_providers/nillion/nillion_action_provider.py

```diff
@@ -0,0 +1,510 @@
+"""Nillion action provider."""
```

Comment:
Could you please add some unit tests for the action provider? They should go [here](https://github.com/coinbase/agentkit/tree/main/python/coinbase-agentkit/tests/action_providers) in a new folder

Reply:
on it

Reply:
done!